spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/ShellCommandsTests.java

```diff
@@ -77,6 +76,8 @@ void singleFileCommand() {
 		assertAppExists("timestamp", ApplicationType.task);
 	}
 
+	//TODO: Boot3x followup
```

Comment:
Thank you for updating both the TODO as well as description. Let's be consistent with approach.